server/data/data-library/src/test/java/org/apache/james/rrt/lib/AbstractRecipientRewriteTableTest.java

```diff
@@ -334,10 +334,46 @@ public void removeGroupMappingShouldDelete() throws Exception {
         virtualUserTable.removeMapping(source, Mapping.group(ADDRESS));
         virtualUserTable.removeMapping(source, Mapping.group(address2));
 
-        assertThat(virtualUserTable.getMappings(USER, domain))
+        assertThat(virtualUserTable.getUserDomainMappings(source))
             .isEqualTo(MappingsImpl.empty());
     }
 
+    @Test
+    public void addAliasMappingShouldStore() throws Exception {
+        Domain domain = Domain.LOCALHOST;
+        String address2 = "test@james";
+        MappingSource source = MappingSource.fromUser(USER, domain);
```

Comment:
This could very well become a SOURCE static final constant shared everywhere in that test. IMO it would make that case easier to read.